### PR TITLE
Creating v1 JNb documentation

### PR DESCRIPTION
**Pull request by michellebieger**
Documentation for twinLab requires example Jupyter Notebooks, as shown here: https://digilab-ai.github.io/twinLab/Python/Examples/

We have a PR up for the Client 2.0 version NBs, as seen here in https://github.com/digiLab-ai/twinLab-Demos/pull/35. 

In order to update the documentation on the twinLab site, we need to update the Markdowns in the twinLab repository (which will be done by myself in subsequent PRs on the twinLab repository), but also keep the v1 NBs in order to link to them through nb viewer. 

This PR copies the current V1 NBs and keeps them in a seperate, tidy directory in this Demo repository.  That should be the only change seen. When PR 35 gets accepted, the NBs outside of this directory will be updated with V2.

### CODE REVIEW

File: poetry.lock
Comment (alexander-mead) [inline poetry.lock:1]: This file shouldn't be in the repository and should be deleted and added to the `.gitignore`.

File: .env.example
Comment (alexander-mead) [inline .env.example:1]: I think we need this file in the repository. Why was it deleted?

File: 00-quickstart.ipynb
Comment (alexander-mead) [inline 00-quickstart.ipynb:1]: Did this change for any reason? If not, it can be reverted.

**Review by alexander-mead — CHANGES_REQUESTED** [review]
Great to store the old notebooks. Some minor changes we can look at after lunch.

File: .env.example
Comment (michellebieger) [reply .env.example:1]: Fixed through the `git checkout` command you sent me. Thank you!

**Comment by michellebieger** [comment]
Updated file changes--thank you! Need to resolve the `poetry.lock` but that is going to be fixed, pending @alexander-mead 's changes.

**Review by alexander-mead — APPROVED** [review]
Looks great!

**Pull request merged**